executor/restore.go

```diff
@@ -58,27 +62,19 @@ func (r *RestoreExecutor) startRestore(restore *k8upv1alpha1.Restore) {
 	j, err := r.buildRestoreObject(restore)
 	if err != nil {
 		r.Log.Error(err, "unable to build restore object")
+		r.setConditionFalse(ConditionJobCreated, "unable to build restore object")
 		return
 	}
 
 	if err := r.Client.Create(r.CTX, j); err != nil {
 		if !apierrors.IsAlreadyExists(err) {
 			r.Log.Error(err, "could not create job")
+			r.setConditionFalse(ConditionJobCreated, "could not create job")
 			return
 		}
 	}
 
-	if err := r.setJobStatusStarted(); err != nil {
-		r.Log.Error(err, "could not update prune status")
-	}
-}
-
-func (r *RestoreExecutor) setJobStatusStarted() error {
-	original := r.Obj
-	new := r.Obj.GetRuntimeObject().DeepCopyObject().(*k8upv1alpha1.Restore)
-	new.GetStatus().Started = true
-
-	return r.Client.Status().Patch(r.CTX, new, client.MergeFrom(original.GetRuntimeObject()))
+	r.setStartedTrue(ConditionJobCreated)
```

Comment:
It would be nice to indicate the id of the created resource. This makes easier to look at that job (no need to do `kubectl get job` and find the matching one from the list).

Apply to other locations where this idea applies.

Reply:
Good idea. Implemented ✔️